### ISSUE TITLE
Yii migrate on PHP 7.4 does not wait for user in put 

### ISSUE THREAD

**Issue by djidji01** [yiisoft/yii2#17782]
### What steps will reproduce the problem?
Yii migrate on PHP 7.4 , after 

> **Apply the above migrations?**

 massage, does not wait for user in put  (yes|no) but it automatically run the default. When I change back php back to 7.3.12 the problem goes away?

### What is the expected result?
Apply migration only on  **yes** input and ignore migration only on **no** input
### What do you get instead?
Migration is automatically not applied if `$default ` parameter of confirm method of  yii\console\Controller `=false` and is automatically applied if I change `$default = true`.
```
public function confirm($message, $default = false)
    {
        if ($this->interactive) {
            return Console::confirm($message, $default);
        }

        return true;
    }
```
![Capture](https://user-images.githubusercontent.com/19886674/71827389-c9699400-30a8-11ea-9b85-f64851deb286.PNG)

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.31?
| PHP version    | 7.4.0  
| Operating system | windows7 64 bit

**Comment by rob006** [comment]
Works fine for me on Linux.

**Comment by samdark** [comment]
That's a bug in 7.4.0 under Windows. It's fixed in 7.4.1.
